src/custom-compiler.ts

```diff
@@ -0,0 +1,15 @@
+import * as __typeof__ts from 'typescript';
+import { TsJestConfig } from './jest-types';
+
+/**
+ * Return a typescript compiler.
+ * Allows config to specify alternative compiler.
+ * For example, `ntypescript`.
+ */
+export function getTypescriptCompiler(config: TsJestConfig): typeof __typeof__ts {
+  if (typeof config.compiler === 'string') {
+    return require(config.compiler);

```

Comment:
Perhaps we should have a catch block for these dynamic requires? Just to give a better error message if it goes wrong.

Reply:
Any ideas what it should say?  Node's default error is:
```
Error: Cannot find module 'ttypescript'
    at Function.Module._resolveFilename (module.js:542:15)
```

Should we wrap it with this prefix?
```
Error: Importing TypeScript compiler: Cannot find module 'ttypescript'
```